addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties

```diff
@@ -0,0 +1,29 @@
+# Binding
+icloud.binding.name=Extension iCloud
+icloud.binding.description=Apple iCloud est utilis� pour obtenir des donn�es telles que le niveau de la batterie ou la localisation d'un ou plusieurs appareils Apple connect�s � un compte iCloud. Les mises � jour sont rapides et pr�cises sans impact notable sur la batterie.
+# Account Thing
+icloud.account-thing.label=Compte iCloud
+icloud.account-thing.description=Ce Bridge vers le compte, repr�sente votre compte Apple iCloud. Vous pouvez en cr�er plusieurs si vous disposez de plusieurs comptes iCloud.
+icloud.account-thing.parameter.apple-id.label=Apple Id
+icloud.account-thing.parameter.apple-id.description=Apple Id (e-mail) permettant l'acc�s aux informations iCloud.
+icloud.account-thing.parameter.password.label=Mot de Passe
+icloud.account-thing.parameter.password.description=Mot de passe Apple iCloud pour l'Apple Id
+icloud.account-thing.parameter.refresh.label=Rafra�chir
+icloud.account-thing.parameter.refresh.description=P�riode de rafra�chissement en minutes.
+icloud.account-thing.property.owner=Propri�taire
+# Device Thing
+icloud.device-thing.label=Equipement iCloud
+icloud.device-thing.description=Ce 'Thing' repr�sente un appareil Apple, tel par exemple un t�l�phone. Il doit �tre connect� � un 'Thing' Compte pour �tre mis � jour. Plusieurs �quipement peuvent �tre rattach�s � un m�me compte.
```

Comment:
Did you check whether the ' ends up correct on the screen, normally quotes have to be escaped with other quotes in property files, as also shown by the failed rendering.

https://www.mscharhag.com/java/resource-bundle-single-quote-escaping

Reply:
Just create a new PR if this really broke it, I will merge now :-)